lesson01.md

```diff
@@ -12,5 +21,587 @@ Lesson 1, Tuesday, 2021-xx-xx
+
+* `"\n"` - new line
+* `"\t"` - tab
+
+
+---
+
+
+### Quiz: Which strings are correct?
+
+```plaintext
+"Hello"
+'World'
+"He said "hello" to me"
+'Let's go!'
+""
+'This is a\nnew line'
+'This is a backslash: \'
+```
+
+---
+
+### Solution
+
+```js
+"Hello"                        // CORRECT
+'World'                        // CORRECT
+"He said "hello" to me";       // WRONG - unescaped quotes
+'Let's go!'                    // WRONG - unescaped '
+""                             // CORRECT
+'This is a\nnew line';         // CORRECT
+'This is a backslash: \';      // WRONG - escaped '
+```
+
+---
+
+The addition operator can be used on Strings and Numbers:
+
+```js
+1 + 1                // is 2
+"hello" + " world"   // is "hello world"
+```
+
+---
+
+### Importance of data types
+
+```js
+1 + 1        // is 2
+"1" + "1"    // is "11"
+```
+
+Make sure to always choose the correct data type for your data,
+because the operators in JavaScript behave differently depending on the data type.
+
+---
+
+
+### Data Type: Boolean
+
+* **Boolean** describes a yes/no situation
+* in JavaScript, we use `true` or `false`
+
+---
+
+
+### DIY?
+
+Open the console, and type in the following:
+
+* true
+* false
+
+Yep, that is it.
+
+It seems not very useful...
+
+YOU JUST WAIT!
+<!-- .element: class="fragment" -->
+
+---
+
+### Comparison operators
+
+For all data types:
+* `===` strict equality
+* `!==` strict inequality
+
+Numbers only:
+* `>` greater than
+* `<` less than
+* `>=` greater or equal
+* `<=` less or equal
+
+All of these return a boolean!
+
+---
+
+### Let's practice
+
+* Are you older than the person next to your right?
+* Do you have the same first name as the person next to you?
+
+---
+
+### Solution
+
+```js
+39 > 36                   // true - I am older
+"Carlo" === "Owen"   // false - not the same first name
+```
+
+---
+
+
+### Logical Operators
+
+These operators also work only on Boolean values:
+
+* `!` Logical NOT
+* `&&` Logical AND
+* `||` Logical OR
+
+We are not going to explain them today, but definitely when we talk about conditionals.
+
+---
+
+### Data Type: Undefined
+
+*  **Undefined** has only one value: `undefined`
+* it is just `undefined`
+* it means there is no value here
+
+---
+
+### Try it out
+
+Now, try to enter some data to the JavaScript console, but think of the data type first:
+
+* Your name
+* Your shoe size
+* Wether you think it's ok to put pineapple on pizza 
+
+---
+
+### Try it out: Solution
+
+```js
+"Carlo"
+45
+false
+```
+
+---
+
+### Summary: what did we learn?
+
+- what is JavaScript?
+<!-- .element: class="fragment"  -->
+- how do we open the JavaScript console in the browser?
+<!-- .element: class="fragment"  -->
+- what can we do in the JavaScript console?
+<!-- .element: class="fragment"  -->
+- what operators have we learnt?
+<!-- .element: class="fragment"  -->
+- what data types have we learnt?
+<!-- .element: class="fragment" -->
+
+---
+
+### Summary: Basic Data types 
+
+- numbers
```

Comment:
do you want to use plural here? `numbers` vs `number` ?

Reply:
ah, not sure. You think singular is better? And in case, shall I also change it for `strings` and `booleans`?